model/metrics/gen-ai.yaml

```diff
@@ -0,0 +1,46 @@
+groups:
+  - id: metric_attributes.gen_ai
+    type: attribute_group
+    brief: 'This group describes GenAI metrics attributes'
+    attributes:
+      - ref: error.type
+        requirement_level:
+          conditionally_required: "if the operation ended in an error"
+        note: |
+          The cardinality of `error.type` SHOULD be low.
+
+          When working across multiple models, it is RECOMMENDED to use a common set of error types.
+
+          Additional details may be captured in domain-specific attributes.
+      - ref: server.address
+        requirement_level: recommended
+      - ref: server.port
+        requirement_level:
+          conditionally_required: If `sever.address` is set.
+      - ref: gen_ai.response.model
+        requirement_level: recommended
+      - ref: gen_ai.request.model
+        requirement_level: required
+      - ref: gen_ai.system
+        requirement_level: required
+      - ref: gen_ai.operation.name
+        requirement_level: required
+  - id: metric.gen_ai.client.token.usage
+    type: metric
+    metric_name: gen_ai.client.token.usage
+    brief: 'Measures number of input and output tokens used'
```

Comment:
[Cohere](https://docs.cohere.com/reference/chat) reports both "used" tokens and "billed" tokens - we should clarify that this is always the billed token count.

Reply:
In the case of Azure OpenAI, customers have the ability to reserve capacity in units (Provisioned Throughput Unit, PTU). In this case, the tokens are not necessarily "billed". It feels like there is a lot of nuance here and perhaps we should leave this general and allow vendor semantics to differentiate?

Reply:
I think we'll need to add a Cohere specific page and provide details on what goes into this metric there. 
I think it can be done in a follow up PR.